playground/terraform/README.md

```diff
@@ -28,14 +28,64 @@ The following items need to be setup for the Playground cluster deployment on GC
 
 # Deployment steps
 
+## 0. Create GCS bucket for state
+
+```bash
+$ gsutil mb -p ${PROJECT_ID} gs://state-bucket-name
+$ gsutil versioning set on gs://state-bucket-name
+```
+
+## 1. Create new environment
+
+To provide informantion about terraform backend, please create:
+
+* New environment folder
+
+```bash
+mkdir environment/env-name
+```
+
+* Backend config
+
+```bash
+echo 'bucket = "put your state bucket name here"' > environment/env-name/state.tfbackend
+```
+
+* Terraform variables config and provide necessary variables
+
+```bash
+touch environment/env-name/terraform.tfvars
+```
+
+Then provide necesarry variables.
```

Comment:
How would someone know the necessary variables?  When I use `*.tfvars` file in the same module as my terraform code by convention both visual studio code and IntelliJ parse the variables file and prompt a developer to fill out the right variables via auto completion.  By having this file in `env-name/terraform.tfvars` it makes it challenging for someone to know what to do unless they run through the solution and copy/paste variable names.